server/test/services/export/AbstractExporterTest.java

```diff
@@ -400,12 +404,14 @@ protected void createFakeProgramWithEnumerator() {
     applicantTwo.save();
     applicationTwo =
         new Application(applicantTwo, fakeProgramWithEnumerator, LifecycleStage.ACTIVE);
+    applicationTwo.setApplicantData(applicantTwo.getApplicantData());
     CfTestHelpers.withMockedInstantNow(
         "2022-02-01T00:00:00Z", () -> applicationTwo.setSubmitTimeToNow());
     applicationTwo.save();
 
     applicationThree =
         new Application(applicantTwo, fakeProgramWithEnumerator, LifecycleStage.OBSOLETE);
+    applicationThree.setApplicantData(applicantTwo.getApplicantData());
```

Comment:
Does an Obsolete application necessarily come from an ACTIVE one? If it can come from a draft maybe this shouldn't have applicant Data

Reply:
> Does an Obsolete application necessarily come from an ACTIVE one?
I believe so, yes

Reply:
> Does an Obsolete application necessarily come from an ACTIVE one?
I believe so, yes